caveclient/chunkedgraph.py

```diff
@@ -709,7 +715,13 @@ def get_operation_details(self, operation_ids: Iterable[int]):
         return r.json()
 
     def get_lineage_graph(
-        self, root_id, timestamp_past=None, timestamp_future=None, as_nx_graph=False
+        self,
+        root_id,
+        timestamp_past=None,
+        timestamp_future=None,
+        as_nx_graph=False,
+        exclude_links_to_future=False,
```

Comment:
can these be timestamps or None? then we could use 2 args instead of 4

Reply:
no this is slightly different, it depends on whether you want to include links that cross over the boundary or not.  For different use cases you might want those or not.  For example, when going backward in time, you might want to know the first root_id that comes before the timestamp_past, because that is the root_id that would be current at timestamp_past.